### ISSUE TITLE
Terminator window not recognized as running application in Ubuntu

### ISSUE THREAD

**Issue by janmyszkier** [gnome-terminator/terminator#276]
steps to reproduce:
- work with Ubuntu 20.04.
- install terminator from apt
- run terminator from "windows key" search
- issue 1: see how the icon does not appear on the applications bar
- issue 2: see how the icon is not displayed along other running applications when pressing the "windows key"

If you minimize all the other apps, the terminator is there, though

**Comment by mattrose** [comment]
I just tried doing this exact thing and it worked as expected.  The icon appeared in the sidebar and in the display of running applications. I was using GNOME.  Please re-open with more information if this persists.

**Comment by Fonfon02** [comment]
Hi, I have exactly the same problem on my Ubuntu 20.04.

Fyi I run:
  - Terminator version 1.91
  - Gnome version 3.36.3

**Comment by janmyszkier** [comment]
@mattrose I did ask someone to check this on fresh ubuntu installation and it indeed works. My ubuntu if an upgraded 18.04 -> 20.04, and I still have the issue regardless of the version I install (from apt or from github). Tried to remove all residual configs, but that didn't help either.
I tried with new user and THAT worked, but I'm unable to find anything in my main HOME dir that could alter this behaviour. Used `grep -ril 'terminal' ~/` but that doesn't yield any results that are interesting. I guess that's a gnome thing and not something related ONLY to terminator. Thank you for checking, anyway. I'll probably re-add my user to fix this as I don't think it makes sense to dig into this more.
